packages/lesswrong/server/updateScores.ts

```diff
@@ -166,12 +173,78 @@ export const batchUpdateScore = async ({collection, inactive = false, forceUpdat
         }
       }
     },
-  ])
+  ]);
+
+  return items.toArray();
+}
+
```

Comment:
I don't love having all this SQL directly in this file, but it's not clear to me what the ideal architecture here would look like yet. In general, I think this file will become a lot simpler post-postgres, and it might make sense to integrate it with `scoring.ts`. The most obvious option would probably be to make some `UpdateScoresRepo`, but I still prefer keeping a 1-to-1 mapping between collections and repos. This will almost certainly be the first place I'll dig into for Postgres-related refactoring once the migrations are done.